grammar/parsing.js

```diff
@@ -206,12 +206,50 @@ class Parsing extends EmbeddedActionsParser {
             return result;
         });
 
+        $.RULE('multiArray', () => {
+            var multiArr = null;
+            let singleArr = null;
```

Comment:
Very minor stuff, but it appears that `multiArr` can be a `const` if you initially declare it as `const multiArr = []`. 

This makes L215 unnecessary and probably also the `this.utils.toArray()`s (even if it wasn't a no-op, which it currently is if you look at `utils.js`)

 Objects and arrays (which are Objects) can almost be declared as `const` unless they are fully reassigned (so the `mutliArr.push()` is OK on a `const`, but `singleArr = $.SUBRULE` needs a `let`, as you have done.)